res/controllers/Denon-MC7000-scripts.js

```diff
@@ -106,76 +109,77 @@ MC7000.PADModePitch = [false, false, false, false];
 
 // PAD Mode Colors
 MC7000.padColor = {
-    'alloff': 0x01,         // typically not needed for PADs
-    'hotcueoff': 0x02,      // lightblue Hot Cue inactive
-    'hotcueon': 0x04,       // darkblue Hot Cue active
-    'sampleroff': 0x27,     // light pink Sampler standard colour
-    'samplerloaded': 0x38,  // dark pink Sampler loaded colour
-    'samplerplay': 0x09,    // green Sampler playing
-    'rollon': 0x10,         // BeatloopRoll active colour
-    'rolloff': 0x1B,        // BeatloopRoll off colour
-    'cueloopon': 0x0D,      // Cueloop colour for activated cue point
-    'cueloopoff': 0x1A      // Cueloop colour inactive
+    "alloff": 0x01,         // switch off completely
+    "hotcueoff": 0x02,      // lightblue Hot Cue inactive
+    "hotcueon": 0x04,       // darkblue Hot Cue active
+    "sampleroff": 0x27,     // light pink Sampler standard colour
+    "samplerloaded": 0x38,  // dark pink Sampler loaded colour
+    "samplerplay": 0x09,    // green Sampler playing
+    "rollon": 0x10,         // BeatloopRoll active colour
+    "rolloff": 0x1B,        // BeatloopRoll off colour
+    "cueloopon": 0x0D,      // Cueloop colour for activated cue point
+    "cueloopoff": 0x1A      // Cueloop colour inactive
 };
 
 /* DECK INITIALIZATION */
-MC7000.init = function () {
-	
+MC7000.init = function() {
+
     // Decks
     MC7000.leftDeck = new MC7000.Deck(1, 3);
     MC7000.rightDeck = new MC7000.Deck(2, 4);
-    
-    // set default Master Volume to give a little room for mixing
-    engine.setValue("[Master]", "gain", 0.85);
-    
+
+    // set default Master Volume to 85% to give a little head room for mixing
+    // engine.setValue("[Master]", "gain", 0.85);
+
     // VU meters
-    engine.connectControl("[Channel1]", "VuMeter", "MC7000.VuMeter");
-    engine.connectControl("[Channel2]", "VuMeter", "MC7000.VuMeter");
-    engine.connectControl("[Channel3]", "VuMeter", "MC7000.VuMeter");
-    engine.connectControl("[Channel4]", "VuMeter", "MC7000.VuMeter");
-        
-    // Platter Ring LED 
+    print(typeof MC7000.VuMeter);
```

Comment:
This was only for test purposes. You can remove it now. To save some time, you can just apply my suggestion directly on github.
```suggestion
```

Reply:
Thanks for pointing this. Yep - deleted.